.github/workflows/push-master.yml

```diff
@@ -39,10 +39,10 @@ jobs:
       run: rm -rf ./dist
 
     - name: Build for PROD environment
-      run: yarn build
+      run: yarn build:staging
```

Comment:
Where do we build prod dist?

Reply:
did not read correctly this conf =/
add commit with fix